src/main/resources/db/migration/V1.3__updateVisit.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE visit
+    ADD COLUMN visit_date_time TIMESTAMP NOT NULL,
+    ADD COLUMN paid VARCHAR(3) NOT NULL,
+    ADD COLUMN description VARCHAR(100) NOT NULL,
+    ADD COLUMN payment VARCHAR(8) NOT NULL
+;
```

Comment:
średnik w nowej lini??

Reply:
poprawię ale - zmiana w pliku nie zrobi problemu póki to jest stary plik z tą samą nazwą?

Reply:
jeżeli lokalnie wyczyścisz sobie tabele flyway to będzie oki, ale pamiętaj że jeżeli będziesz pracować z kimś jeszcze to takie rzeczy dodajesz przez kolejny skrypt/update

Reply:
poprawione wspólnie